examples/notebook_tutorial/ebonite_tutorial.ipynb

```diff
@@ -36,44 +36,45 @@
+      "Requirement not upgraded as not directly required: six>=1.10.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from flasgger==0.9.3) (1.12.0)\n",
+      "Requirement not upgraded as not directly required: scipy>=0.17.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from scikit-learn) (1.1.0)\n",
+      "Requirement not upgraded as not directly required: numpy>=1.11.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from scikit-learn) (1.18.2)\n",
+      "Requirement not upgraded as not directly required: joblib>=0.11 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from scikit-learn) (0.13.1)\n",
+      "Requirement not upgraded as not directly required: gitdb2>=2.0.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from GitPython==3.0.3->ebonite) (2.0.5)\n",
+      "Requirement not upgraded as not directly required: MarkupSafe>=0.23 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from Jinja2==2.10.1->ebonite) (1.0)\n",
+      "Requirement not upgraded as not directly required: websocket-client>=0.32.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages/websocket_client-0.56.0-py3.6.egg (from docker==4.1.0->ebonite) (0.56.0)\n",
+      "Requirement not upgraded as not directly required: certifi>=2017.4.17 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from requests==2.22.0->ebonite) (2019.6.16)\n",
+      "Requirement not upgraded as not directly required: idna<2.9,>=2.5 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from requests==2.22.0->ebonite) (2.8)\n",
+      "Requirement not upgraded as not directly required: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from requests==2.22.0->ebonite) (1.24.3)\n",
+      "Requirement not upgraded as not directly required: chardet<3.1.0,>=3.0.2 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from requests==2.22.0->ebonite) (3.0.4)\n",
+      "Requirement not upgraded as not directly required: smmap2>=2.0.0 in /Users/mike0sv/miniconda3/envs/py36/lib/python3.6/site-packages (from gitdb2>=2.0.0->GitPython==3.0.3->ebonite) (2.0.5)\n",
+      "\u001b[31mpyasn1-modules 0.2.7 has requirement pyasn1<0.5.0,>=0.4.6, but you'll have pyasn1 0.4.4 which is incompatible.\u001b[0m\n",
+      "\u001b[31mawscli 1.16.71 has requirement PyYAML<=3.13,>=3.10, but you'll have pyyaml 5.1.2 which is incompatible.\u001b[0m\n",
+      "\u001b[33mYou are using pip version 10.0.1, however version 20.0.2 is available.\n",
+      "You should consider upgrading via the 'pip install --upgrade pip' command.\u001b[0m\n"
      ]
     }
    ],
    "source": [
-    "! pip install -U ebonite flask flasgger scikit-learn"
+    "! pip install -U ebonite flask flasgger==0.9.3 scikit-learn"
```

Comment:
I hope to eliminate the need to install `flask` and  `flasgger` here in EBNT-270.

Reply:
I mean for building Docker images, for `run_model_server` there is of course no way around it.

Reply:
i thought we will use base images just for testing... but that is topic of another PR